app/models.py

```diff
@@ -28,3 +28,16 @@ def get_by_email(email: str) -> User:
     @staticmethod
     def get_by_user_id(user_id: int) -> User:
         return User.query.get(user_id)
+
+
+class Post(db.Model):
+    __tablename__ = 'post'
+
+    post_id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.user_id'), nullable=False)
+    image_url = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.String(255), nullable=False)
```

Comment:
I think we should increase this amount closer to the 2K that Instagram allows since it is common for posts to have walls of text.